src/Mutator/ProfileList.php

```diff
@@ -464,6 +464,9 @@ final class ProfileList
         // Extensions
         'BCMath' => Mutator\Extensions\BCMath::class,
         'MBString' => Mutator\Extensions\MBString::class,
+
+        // Internal only usage
```

Comment:
`Internal usage only`

Reply:
fixed in https://github.com/infection/infection/pull/1557